device/bgp/bgpsrteadvanced.yaml

```diff
@@ -0,0 +1,25 @@
+components:
+  schemas:
+    BgpSrTe.Advanced:
+      description: >-
+        Configuration for advanced BGP route range settings.
+      type: object
+      properties:
+        origin:
+          description: >-
+            The origin value. 
+            A null value means the origin feature is not enabled.
+          type: string
+          enum:
+          - igp
+          - egp
+        multi_exit_discriminator:
+          description: >-
+            The multi exit discriminator (MED) value. 
+            A null value means the MED feature is not enabled.
+          type: integer
+        local_preference:
```

Comment:
Since default mode for BGP is iBGP probably this should have a default value to have the default iBGP use-case working. This is ignored by default for eBGP but there should be a choice provided to not include it. For this a separate related field 'enable_local_preference' , default value 'true' , can be provided.

Reply:
done